kolibri/plugins/learn/assets/src/views/BookmarkPage.vue

```diff
@@ -76,41 +57,28 @@
       };
     },
     components: {
-      ContentCard,
+      ContentCardGroupGrid,
     },
     mixins: [commonCoreStrings, responsiveWindowMixin],
     data() {
       return {
         loading: true,
         bookmarks: [],
         more: null,
-        resourcePanelNode: null,
-        now: now(),
       };
     },
     computed: {
-      infoText() {
-        return this.coreString('viewInformation');
-      },
-      removeText() {
-        return this.coreString('removeFromBookmarks');
+      footerIcons() {
+        return { close: 'removeFromBookmarks', info: 'viewInformation' };
       },
     },
     created() {
       ContentNodeResource.fetchBookmarks({ params: { limit: 25 } }).then(data => {
         this.more = data.more;
-        this.bookmarks = data.results;
+        this.bookmarks = data.results.map(normalizeContentNode);
```

Comment:
Although it's possible that API returns an empty array in `data.results` and therefore `map` won't fail even when there are no bookmarks, I'd suggest adding a guard to check if `data.results` are truthy before trying to use `map` just in case it was `undefined` or `null`, just to be sure. Something like

```
this.bookmarks = data.results ? data.results.map(normalizeContentNode) : []
```